video/transcoder/job_test.py

```diff
@@ -132,6 +136,36 @@ def test_create_job_from_preset(capsys, test_bucket):
     assert "Deleted job" in out
 
 
+def test_create_job_from_preset_batch_mode(capsys, test_bucket):
+    create_job_from_preset_batch_mode.create_job_from_preset_batch_mode(
+        project_id, location, input_uri, output_uri_for_preset_batch_mode, preset
+    )
+    out, _ = capsys.readouterr()
+    job_name_prefix = f"projects/{project_number}/locations/{location}/jobs/"
+    assert job_name_prefix in out
+
+    str_slice = out.split("/")
+    job_id = str_slice[len(str_slice) - 1].rstrip("\n")
+    job_name = f"projects/{project_number}/locations/{location}/jobs/{job_id}"
+    assert job_name in out
+
+    get_job.get_job(project_id, location, job_id)
+    out, _ = capsys.readouterr()
+    assert job_name in out
+
+    time.sleep(30)
+
+    _assert_job_state_succeeded_or_running(capsys, job_id)
+
+    list_jobs.list_jobs(project_id, location)
+    out, _ = capsys.readouterr()
+    assert job_name in out
```

Comment:
ditto

Reply:
I was not able to find a better way of doing this. Other samples in the repo use the same method of testing list operations.

Reply:
Ok. Thanks. It's a new approach so some of older samples may not use it yet.